### PR TITLE
_

### PR DESCRIPTION
**Pull request by smmr0**

### CODE REVIEW

File: jeremiahmegel-pella.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Pella, IA
+
+### Places to eat
+- CUIFS
+- Fred's
```
Comment (bjswick33) [inline jeremiahmegel-pella.md]: @jeremiahmegel Is this a local joint?  What's good there?

Comment (jeremiahmegel) [inline jeremiahmegel-pella.md]: It's the restaurant in the student center at our college. My favorite thing is the buffalo chicken wrap.

**Comment by bjswick33** [comment]
Buffalo chicken sounds good right now.

**Comment by jeremiahmegel** [comment]
That it does. Especially since I'm skipping lunch for this training. :poultry_leg:

**Comment by bjswick33** [comment]
I think this looks good!  Ready to :shipit: and get some :chicken: !

**Pull request closed without merging**